libraries/chain/chain_controller.cpp

```diff
@@ -241,6 +241,8 @@ processed_transaction chain_controller::_push_transaction(const signed_transacti
    if (!_pending_tx_session.valid())
       _pending_tx_session = _db.start_undo_session(true);
 
+   FC_ASSERT( _pending_transactions.size() < 1000, "too many pending transactions, try again later" );
```

Comment:
If we make 1000 a config parameter in chain_controller, then we can adjust these during testing and not have to wait for a tag and recompile for the test net

Reply:
@brianjohnson5972 go ahead and prepare a PR with making these two configurable.